test/src/ui/widget.spec.ts

```diff
@@ -0,0 +1,1792 @@
+/*-----------------------------------------------------------------------------
+| Copyright (c) 2014-2016, PhosphorJS Contributors
+|
+| Distributed under the terms of the BSD 3-Clause License.
+|
+| The full license is in the file LICENSE, distributed with this software.
+|----------------------------------------------------------------------------*/
+'use strict';
```

Comment:
use strict is no longer required as of TS 1.8 - it gets added automatically by the compiler